R/lifecycle.R

```diff
@@ -0,0 +1,43 @@
+#' Life cycle of the rlang package
+#'
+#' The rlang package is currently maturing. Unless otherwise stated,
+#' this applies to all its exported functions. See [lifecycle-stable]
+#' for a list of functions marked as stable.
+#'
+#' @name lifecycle
+NULL
+
+#' Stable functions
```

Comment:
Might be better to documents these all one file? (With sections for different phases)